backend/src/apiserver/template/argo_template.go

```diff
@@ -93,6 +93,12 @@ type Argo struct {
 	wf *util.Workflow
 }
 
+func (t *Argo) IsCacheEnabled() bool {
+	return true
```

Comment:
Does this mean that v1 pipelines always have the cache enabled? If so, I think we should be sure to document that when we document this new configuration option.